src/2023/index.html

```diff
@@ -378,13 +378,21 @@ <h2>Partners</h2>
                   </svg>
                 </div>
               </a>
+
+              <a href="https://www.linkedin.com/events/helveticruby20237121673048570634240/comments/" title="Follow us on LinkedIn">
```

Comment:
I don't have LinkedIn. The link takes me to a "Page not found", I expected it to be "You need to sign in". Can you confirm that the link is valid?

Reply:
Ridiculous LinkedIn, I tried it yesterday, and I copied the link from a different place (the original share button) today and of course I tested it. It doesn't seem to work reliably. Will look into it. 

Can you try this one in the meantime? https://www.linkedin.com/events/helveticruby20237121673048570634240/

Reply:
I see this with the new link:

![image](https://github.com/MINASWAN/helvetic-ruby.ch/assets/677998/6c920937-cfcc-4b98-9573-d3e11784aa21)

Reply:
I get the same.

Reply:
Nice.